src/AutomatedCar/App.xaml.cs

```diff
@@ -36,21 +36,51 @@ public override void OnFrameworkInitializationCompleted()
                 PolylineGeometry geom = new PolylineGeometry(points, false);
 
                 World world = World.Instance;
-                world.PopulateInstance("config.json");
 
+                world.PopulateInstance("config.json");
+                world.VisibleWidth = 960;
+                world.VisibleHeight = 720;
                 world.Width = 2000;
                 world.Height = 1000;
-
-                Circle circle = new Circle(400, 200, "circle.png", 20);
+ 
+                Circle circle = new Circle(0, 0, "circle.png", 20);
                 circle.Width = 40;
                 circle.Height = 40;
                 circle.ZIndex = 2;
                 world.AddObject(circle);
 
-                AutomatedCar controlledCar = new Models.AutomatedCar(50, 50, "car_1_white.png");
+                Circle circle2 = new Circle(2000-40, 1000-40, "circle.png", 20);
+                circle2.Width = 40;
+                circle2.Height = 40;
+                circle2.ZIndex = 2;
+                world.AddObject(circle2);
+
+                Circle circle3 = new Circle(2000-40, 0, "circle.png", 20);
+                circle3.Width = 40;
+                circle3.Height = 40;
+                circle3.ZIndex = 2;
+                world.AddObject(circle3);
+                Circle circle4 = new Circle(0, 1000-40, "circle.png", 20);
+                circle4.Width = 40;
+                circle4.Height = 40;
+                circle4.ZIndex = 2;
+                world.AddObject(circle4);
+
+                Circle circle5 = new Circle(900, 550, "circle.png", 20);
+                circle5.Width = 40;
+                circle5.Height = 40;
+                circle5.ZIndex = 2;
+                world.AddObject(circle5);
+
+                 Circle circle6 = new Circle(900, 150, "circle.png", 20);
+                circle6.Width = 40;
+                circle6.Height = 40;
+                circle6.ZIndex = 2;
+                world.AddObject(circle6);
```

Comment:
Ezek a `Circle` duplikációk relevánsak?
Ezt tehetnéd egy listába és a létrehozást is csinálhatod ciklussal, úgy szebb 😃

Reply:
Nem túl releváns, csak ott felejtettem. Törlöm.